collect_app/src/main/java/org/odk/collect/android/provider/FormsProvider.java

```diff
@@ -530,10 +531,10 @@ public int update(Uri uri, ContentValues values, String where,
     // https://developer.android.com/reference/android/content/UriMatcher
```

Comment:
Remove comment from here?

Reply:
Or maybe I should remove the comment from UriUtils. The method name explains 'what' we are trying to do, but the comment explains 'why'. The 'why' part seems like it should be kept closer to the decision point, i.e. in FormsProvider. So should I remove the identical comment from UriUtils instead?

Reply:
Ah, yes, I see. I agree that what you suggest makes more sense. 👍